### ISSUE TITLE
When is your most productive time of the day (coding-wise, outside job hours)?

### ISSUE THREAD

**Issue by revelt** [kevinjalbert/ama#1]
From your past experience, when do you achieve the most coding-wise (outside the job hours)?
Cheers!

**Comment by kevinjalbert** [comment]
Hello @revelt,

I really want to be a morning person and have those early hours as dedicated productivity time... But I'm not haha.

I normally slot time after dinner when my son (11 months at the time of writing this) is asleep. Generally, 8pm-11pm is the time I can be productive outside of work hour.

If I'm lucky and have some part of the day open on the weekend I'll try to slot in something. The biggest issue I have for _productive time_ is to have a solid block of it (i.e., 1+ hour). In addition, family time is during the daytime, so I mostly have to be productive in the evenings. Of course, even those evenings can be booked up with other events and obligations.

In short, my most productive time of the day coding-wise is when I have free time in the evenings or weekends. 😆 

Thanks for the question!
